components/avatars/avatars.css

```diff
@@ -0,0 +1,26 @@
+
+.avatar{
+    width: 4rem;
+    height: 4rem;
+    /* border: solid; */
+    /* border-radius: 50%; */
+}
+.avatar>img{
+    border-radius: 50%;
+}
+.avatar.sm{
```

Comment:
you could directly use .sm without .avatar

Reply:
Done with replacing these classes with much better named classes